Lib/unittest/test/testmock/testmock.py

```diff
@@ -407,6 +407,14 @@ def test_assert_called_once_with(self):
             lambda: mock.assert_called_once_with('bob', 'bar', baz=2)
         )
 
+    def test_assert_called_once_with_call_list(self):
+        m = Mock()
+        m(1)
+        m(2)
+        self.assertRaisesRegex(AssertionError,
+            r"Calls: \[call\(1\), call\(2\)\]",
```

Comment:
You may use re.escape() instead. Or maybe use "with assertRaises(AssertionError) as cm: ...; self.assertIn("Calls: ...", str(cm.exception))".